### PR TITLE
migrate critical constants and acentric factors batch 3

### PR DESCRIPTION
**Pull request by saikat-samanta**

### CODE REVIEW

File: src/data/critical-constants-acentric-factor.ts [hunk 1]
```diff
@@ -303,4 +303,912 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    name: 'Chloromethane',
+    formula: 'CH3Cl',
+    CAS: '74-87-3',
+    molecularWeight: 50.4875,
+    criticalTemperature: 416.25,
+    criticalPressure: 6.68,
+    criticalVolume: 0.141,
+    criticalCompressibilityFactor: 0.272,
+    acentricFactor: 0.151,
+  },
```
Comment (sumitnegi7) [inline src/data/critical-constants-acentric-factor.ts:674]:
```suggestion
  },
  
```

New lines missing

File: src/data/critical-constants-acentric-factor.ts [hunk 2]
```diff
@@ -303,4 +303,912 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    CAS: '7778-85-0',
+    molecularWeight: 104.14758,
+    criticalTemperature: 543,
+    criticalPressure: 3.446,
+    criticalVolume: 0.35,
+    criticalCompressibilityFactor: 0.267,
+    acentricFactor: 0.352222,
+  },
+  DimethylAcetylene: {
+    name: ' Dimethyl acetylene',
```
Comment (qur786) [inline src/data/critical-constants-acentric-factor.ts:1204]:
```suggestion
    name: 'Dimethyl acetylene',
```

File: src/data/critical-constants-acentric-factor.ts [hunk 3]
```diff
@@ -303,4 +303,912 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    name: 'Chlorobenzene',
+    formula: 'C6H5Cl',
+    CAS: '108-90-7',
+    molecularWeight: 112.5569,
+    criticalTemperature: 632.35,
+    criticalPressure: 4.5191,
+    criticalVolume: 0.308,
+    criticalCompressibilityFactor: 0.265,
+    acentricFactor: 0.249857,
+  },
```
Comment (qur786) [inline src/data/critical-constants-acentric-factor.ts:641]:
```suggestion
  },
  
```
newline missing , and after this compound as well

**Review by qur786 — CHANGES_REQUESTED** [review]
All data is correct but have some newline and whitespace issues.

**Review by qur786 — COMMENTED** [review]
All good

**Review by qur786 — APPROVED** [review]

**Review by maneetgoyal — APPROVED** [review]
👍

**Pull request merged**